.github/workflows/ci-windows.yml

```diff
@@ -2,13 +2,11 @@
 #
 #   * The os matrix consists of windows-latest only.
 #   * Installs Visual Studio in the os image.
-#   * Doesn't install cmake 3.13.x, because only recent versions of cmake with
-#     the following bug fix detect Visual Studio compilers correctly:
+#   * Doesn't install cmake 3.17.x, because only cmake 3.18 or later, with the
+#     following bug fix, detects Visual Studio compilers correctly:
```

Comment:
I updated this comment with the cmake version that `ci.yml` installs now. I also added the first cmake version that has the bug fix we need on Windows.